### PR TITLE
Added Twitter link to social media section in the footer

### PR DESCRIPTION
**Pull request by Khatwangadhareddy**
what is this PR about ?
Add new social media icon #249
Screenshot 2024-10-07 161715

### CODE REVIEW

File: delivery.html [hunk 1]
```diff
@@ -182,8 +182,10 @@ <h4 style="margin-top: 15px;">Follow Us</h4>
                 <ul class="links">
                     <li><a href="#" class="social"><i class="fa-brands fa-facebook-f"></i> Facebook</a></li>
                     <li><a href="#" class="social"><i class="fa-brands fa-instagram"></i> Instagram</a></li>
-                    <li><a href="#" class="social"><i class="fa-brands fa-linkedin-in"></i> Linkedin</a></li>
+                    <li><a href="#" class="social"><i class="fa-brands fa-linkedin-in"></i> LinkedIn</a></li>
+                    <li><a href="#" class="social"><i class="fa-brands fa-twitter"></i> Twitter</a></li> <!-- Added Twitter -->
```
Comment (akash70629) [inline delivery.html:186]: The Twitter icon has been updated. Please replace the existing icon with the latest version to ensure consistency.

File: dine-in.html [hunk 2]
```diff
@@ -229,8 +229,10 @@ <h4 style="margin-top: 15px;">Follow Us</h4>
                 <ul class="links">
                     <li><a href="#" class="social"><i class="fa-brands fa-facebook-f"></i> Facebook</a></li>
                     <li><a href="#" class="social"><i class="fa-brands fa-instagram"></i> Instagram</a></li>
-                    <li><a href="#" class="social"><i class="fa-brands fa-linkedin-in"></i> Linkedin</a></li>
+                    <li><a href="#" class="social"><i class="fa-brands fa-linkedin-in"></i> LinkedIn</a></li>
+                    <li><a href="#" class="social"><i class="fa-brands fa-twitter"></i> Twitter</a></li> <!-- Added Twitter -->
```
Comment (akash70629) [inline dine-in.html:233]: The Twitter icon has been updated. Please replace the existing icon with the latest version to ensure consistency.

File: index.html [hunk 3]
```diff
@@ -489,8 +489,10 @@ <h4 style="margin-top: 15px;">Follow Us</h4>
                 <ul class="links">
                     <li><a href="#" class="social"><i class="fa-brands fa-facebook-f"></i> Facebook</a></li>
                     <li><a href="#" class="social"><i class="fa-brands fa-instagram"></i> Instagram</a></li>
-                    <li><a href="#" class="social"><i class="fa-brands fa-linkedin-in"></i> Linkedin</a></li>
+                    <li><a href="#" class="social"><i class="fa-brands fa-linkedin-in"></i> LinkedIn</a></li>
+                    <li><a href="#" class="social"><i class="fa-brands fa-twitter"></i> Twitter</a></li> <!-- Added Twitter -->
```
Comment (akash70629) [inline index.html:493]: The Twitter icon has been updated. Please replace the existing icon with the latest version to ensure consistency.

**Review by akash70629 — CHANGES_REQUESTED** [review]
Please change the highlighted sections .

File: delivery.html (hunk 1 above)
Comment (Khatwangadhareddy) [reply delivery.html:186]: ok I will do it

Comment (akash70629) [reply delivery.html:186]: I appreciate it bro!

**Comment by DharshiBalasubramaniyam** [comment]
@Khatwangadhareddy @akash70629 the issue is not created by you both. Hence I can't assign you the issue.

@akash70629, how can you review the pr in my repo?

**Pull request closed without merging**